src/controllers/tests/users.test.js

```diff
@@ -27,14 +27,14 @@ describe('usersController tests', () => {
     boom: {
       badRequest: jest.fn()
     },
-    json: payload => payload
+    json: (payload) => payload
   };
 
-  it('should return user', async () => {
+  /* it('should return user', async () => {
```

Comment:
Remove this test instead comment. Also add tests for all requests: createUser, updateUser, deleteUser, getUsers